extension/package.json

```diff
@@ -1360,24 +1360,9 @@
       },
       {
         "view": "dvc.views.welcome",
-        "contents": "New to the extension?\n[Show Walkthrough](command:dvc.getStarted)\n\n",
+        "contents": "New to the extension?\n[Show Walkthrough](command:dvc.getStarted)\n\nThe extension is currently unable to initialize.\n[Show Setup](command:dvc.showSetup)",
         "when": "true"
       },
-      {
-        "view": "dvc.views.welcome",
-        "contents": "DVC is currently unavailable, [learn more](https://dvc.org/doc).\n[Setup the Workspace](command:dvc.setupWorkspace)",
-        "when": "!dvc.commands.available && !dvc.cli.incompatible"
-      },
-      {
-        "view": "dvc.views.welcome",
-        "contents": "The located version of DVC is incompatible with the extension.\n[Find Another](command:dvc.setupWorkspace)\nUpgrade DVC then...\n[Check Compatibility](command:dvc.checkCLICompatible)",
```

Comment:
[F] I moved the `Check Compatibility` button inside of the webview so that users don't have to reload the entire project if they have an outdated version of the global CLI.